v_min: 0
b_max: 160
h_max: 0
s_min: 0
b_min: 42
h_min: 0
g_max: 160
g_min: 44
s_max: 0
v_max: 0
r_max: 255
r_min: 200